control-services/bin/setup.sh

```diff
@@ -22,6 +22,20 @@ SSH_AUTH_KEYS_PATH="/home/control/.ssh/authorized_keys"
 sudo apt-get update -y
 sudo apt-get upgrade -y
 
+# Install go for building semaphore
+sudo rm -rf /usr/local/go
+wget https://go.dev/dl/go1.17.7.linux-amd64.tar.gz -P /tmp/
+sudo tar -C /usr/local/ -xzf /tmp/go1.17.7.linux-amd64.tar.gz
+
+go version
+
+# Install task
```

Comment:
We dont need to install task on control?  I thought this was one of the major problems we encountered?

Reply:
It does get installed, but in `semaphore.py`. I'll clean this up.